services/src/modules/registry-schema/helpers/index.ts

```diff
@@ -0,0 +1,3 @@
+import proceedPolicy from './policy-helper';
```

Comment:
Not a fan of the`proceedPolicy` name here, the function does a very specific function but the name is quite generic. How about `markOptionalPolicyArgs`?

Regarding the file, it is named `policy-helper` but exports some specific utility function as default. We should either rename the file specifically after that function, or change the export to be non-default